src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -130,6 +130,35 @@ static void append_topic_properties(
     };
 }
 
+static void log_topic_status(std::vector<cluster::topic_result> c_res) {
+    // Group-by error code, value is list of topic names as strings
+    absl::flat_hash_map<cluster::errc, std::vector<ss::sstring>> err_map;
```

Comment:
suggestion: Could we store the topic names as a `topic_namespace_view` so we don't have to allocate a string for each one?

Reply:
I didn't do that because then that would require a future map() to convert the list of topic_namespace_views to strings before call to `boost::algorithm::join` although i suppose we could write a custom method to just join using plain string concatenation, but i preferred using out of the box utils

Reply:
ah I knew you had a good reason then :)

Reply:
actually (just double checked) `boost::algorithm::join` necessitates this but `fmt::join` actually will implicitly convert to string for you. So will push fix